docs/tutorials/using_apis/connectivity/light-control.md

```diff
@@ -162,32 +172,33 @@ connected. IP = 2001:db8:a0b:12f0::1
 You can use this IP address to `ping` from your PC and verify that the connection is working correctly.
 
 ## Memory optimizations
+<!--there's no clear flow to this tutorial. Some of it is steps I need to take, and some of it is background info, but the two are not clearly flagged-->
 
-On some limited platforms, for example NCS36510 or KW24D, building this application might run out of RAM or ROM. In those cases, you can try these instructions to optimize the memory use.
+On some limited platforms, for example NCS36510 or KW24D, running this application might cause the device to run out of RAM or ROM. In those cases, you can try optimizing memory use.
 
-### Mbed TLS configuration
+### Use a customized Mbed TLS configuration
 
-You can set the custom Mbed TLS configuration by adding `"macros": ["MBEDTLS_USER_CONFIG_FILE=\"mbedtls_config.h\""]` to the `.json` file. The [example Mbed TLS configuration](https://github.com/ARMmbed/mbed-os-example-mesh-minimal/blob/master/mbedtls_config.h) minimizes the RAM and ROM use of the application. The is not guaranteed to work on every Mbed Enabled platform.
+The [example Mbed TLS configuration](https://github.com/ARMmbed/mbed-os-example-mesh-minimal/blob/master/mbedtls_config.h) minimizes the RAM and ROM use of the application; it saves you 8.7 kB of RAM but uses 6.8 kB of additional flash. This is not guaranteed to work on every Mbed Enabled platform.<!--what does it do? and why isn't the smaller version the default? what are its disadvantages?-->
```

Comment:
@SeppoTakalo @jarlamsa Can you answer this?

Reply:
@mikter @artokin please comment

Reply:
The mesh-minimal application needs to be compiled for multiple targets and we don't have generic (mesh-specific) mbed-tls configuration file that can be used for all targets. Therefore mbed-tls default configuration file is used by default.